DS/Session3/priorityqueue/PriorityQueueTest.java

```diff
@@ -0,0 +1,61 @@
+package priorityqueue;
+
+import java.util.Scanner;
+
+public class PriorityQueueTest {
+	
+	public static void main(String[] args)
+    {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Priority Queue Test\n");   
+ 
+        System.out.println("Enter size of priority queue ");
+        PriorityQueu pq = new PriorityQueu(scan.nextInt() );
+ 
+        char ch;
+        /*  Perform Priority Queue operations */
+        do    
+        {
+            System.out.println("\nPriority Queue Operations\n");
+            System.out.println("1. insert");
+            System.out.println("2. remove");
+            System.out.println("3. check empty");
+            System.out.println("4. check full");
+            System.out.println("5. clear");
+            System.out.println("6. size");
+ 
+            int choice = scan.nextInt();            
+            switch (choice) 
+            {
+            case 1 : 
+                System.out.println("Enter job name and priority");
+                pq.insert(scan.next(), scan.nextInt() );                     
+                break;                          
+            case 2 : 
+                System.out.println("\nJob removed \n\n"+ pq.remove()); 
+                break;        
+            case 3 : 
+                System.out.println("\nEmpty Status : "+ pq.isEmpty() );
+                break; 
+            case 4 : 
+                System.out.println("\nFull Status : "+ pq.isFull() );
+                break; 
+            case 5 : 
+                System.out.println("\nPriority Queue Cleared");
+                pq.clear();
```

Comment:
But, this can be done in one line because we don't need a variable to store the value first then printing the result in a separate statement. So, if i store the value first in a separate variable then only it's possible to do.